.github/workflows/reusable-scala-steward.yml

```diff
@@ -0,0 +1,40 @@
+name: Reusable Scala Steward workflow
+
+on:
+  workflow_call:
+    inputs:
+      app_id:
+        required: true
+        type: string
+    secrets:
+      private_key:
+        required: true
+
+jobs:
+  scala-steward:
+    name: scala-steward
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo for REPOSITORIES.md (which lists either public or private repos)
+        uses: actions/checkout@v3
+      - name: Checkout guardian/scala-steward-public-repos for our shared standard scala-steward.conf
+        uses: actions/checkout@v3
+        with:
+          repository: guardian/scala-steward-public-repos
+          path: common-config
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ inputs.app_id }}
+          private_key: ${{ secrets.private_key }}
+
+      - name: Execute Scala Steward
+        uses: scala-steward-org/scala-steward-action@v2.18.0
+
+        with:
+          # Use the token generated by the previous step and the 
+          # Guardian Scala Steward GitHub App
+          github-token: ${{ steps.generate-token.outputs.token }}
+          repos-file: REPOSITORIES.md
+          repo-conf: common-config/scala-steward.conf # from checkout of guardian/scala-steward-public-repos
```

Comment:
Whoops, this should have been `repo-config`, not `repo-conf`, what am I like.

https://github.com/scala-steward-org/scala-steward-action#configuration
https://github.com/guardian/scala-steward-public-repos/runs/7566166055?check_suite_focus=true#step:5:1